.github/workflows/build.yml

```diff
@@ -115,6 +116,69 @@ jobs:
           leeway build dev/preview/previewctl:docker -Dversion="${{needs.configuration.outputs.version}}"
           echo "previewctl_hash=$(leeway describe dev/preview/previewctl:docker -Dversion="${{needs.configuration.outputs.version}}" -t '{{ .Metadata.Version }}')" >> $GITHUB_OUTPUT
 
+  update_pr_desc:
+    name: Update PR summary
+    needs: [ configuration ]
+    if: needs.configuration.outputs.pr_number != ''
+    concurrency:
+      group: ${{ github.head_ref || github.ref_name }}-update-pr-desc
+    runs-on: [ self-hosted ]
+    container:
+        image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:kylos101-prebuild-jam-gha.5832
+        volumes:
+          - /var/tmp:/var/tmp
+          - /tmp:/tmp
+    steps:
+      - uses: actions/checkout@v3
+      - name: Configure workspace
+        run: |
+          sudo chown -R gitpod:gitpod /__t
+          # Needed by docker/login-action
+          sudo chmod goa+rw /var/run/docker.sock
+      - name: Set outputs
+        id: outputs
+        shell: bash
+        env:
+          HOME: /home/gitpod
+          PREVIEW_ENV_DEV_SA_KEY: ${{ secrets.GCP_CREDENTIALS }}
+        run: |
+          # Authenticate with GCP so we can use the Leeway cache
+          export PREVIEW_ENV_DEV_SA_KEY_PATH="$HOME/.config/gcloud/preview-environment-dev-sa.json"
+          echo "${PREVIEW_ENV_DEV_SA_KEY}" > "${PREVIEW_ENV_DEV_SA_KEY_PATH}"
+          gcloud auth activate-service-account --key-file "${PREVIEW_ENV_DEV_SA_KEY_PATH}"
+
+          export LEEWAY_WORKSPACE_ROOT="$(pwd)"
+          leeway run dev/preview/previewctl:install
+
+          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+
+          summary=$(previewctl report | base64)
```

Comment:
it is because is report is multi line html it has to be escaped and served as multiline output